go/apps/simpleexample/SimpleDomain/copybins

```diff
@@ -1,9 +1,9 @@
 #
-
```

Comment:
i think defines and copybins should read from GOPATH env variable (currently assumes ~/bin is same as $GOATPH/bin)